Dockerfile.ci-rp

```diff
@@ -5,6 +5,7 @@ ARG VERSION
 # builder is responsible for all compilation and validation of the RP
 ###############################################################################
 FROM ${REGISTRY}/ubi8/go-toolset:1.20.12-5 as builder
+LABEL builder=true
```

Comment:
Small request: could you prefix the label names with something like "aro-"? I think the use of labels here makes sense, but I think this small change will reduce the likelihood of a collision with other images.

Reply:
Yes even I thought about the same when I released this draft pr but will complete the mentioned changes in following commits